tests/unit/test_query.py

```diff
@@ -1342,7 +1384,16 @@ def test_from_api_repr_w_struct_type(self):
             _make_subparam("name", "STRING", "Bharney Rhubbyl"),
             _make_subparam("age", "INT64", 31),
         )
-        self.assertEqual(param.array_type, "STRUCT")
+
+        self.assertEqual(
+            str(param.array_type),
+            str(
+                StructQueryParameterType(
+                    ScalarQueryParameterType("STRING", name="name"),
+                    ScalarQueryParameterType("INT64", name="age"),
+                )
+            ),
+        )
```

Comment:
It doesn't work to do `assertEqual(param.array_type, StuctQueryParameterType(Sclar...)`. Not sure what the better way would be.